src/App.tsx

```diff
@@ -109,41 +105,19 @@ export const App = () => {
     [],
   );
 
-  const isMobileLayout = useIsMobileLayout();
-  const showMobileSideBar = isMobileLayout && isMobileSideBarOpen;
-  const showDesktopSideBar = !isMobileLayout && isDesktopSideBarOpen;
-  const sideBarStyles = `${styles.sidebar} ${isMobileSideBarOpen ? styles.isOpen : ''} ${isDesktopSideBarOpen ? '' : styles.isHidden}`;
-
   return (
     <div className={styles.app}>
-      <div className={sideBarStyles}>
-        <div className={styles.sidebarHeader}>
-          <img src={hardDiamondLogo} alt="Hard Diamond logo" height={40} />
-          <div className={styles.buttonGroup}>
-            {showMobileSideBar && (
-              <MobileSideBar
-                onCloseClick={() => setIsMobileSideBarOpen(false)}
-              />
-            )}
-            {showDesktopSideBar && (
-              <DesktopSideBar
-                onCloseClick={() => setIsDesktopSideBarOpen(false)}
-              />
-            )}
-          </div>
-        </div>
-
-        <div className={styles.sidebarContent}>
-          <ChatHistory
-            chatHistories={conversationHistories}
-            onSelectConversation={setActiveConversationId}
-            activeConversationId={activeConversationId}
-            onDeleteConversation={handleDeleteConversation}
-            onUpdateConversationTitle={handleUpdateConversationTitle}
-          />
-        </div>
-      </div>
-
+      <SideBar
+        conversationHistories={conversationHistories}
+        onSelectConversation={setActiveConversationId}
+        activeConversationId={activeConversationId}
+        onDeleteConversation={handleDeleteConversation}
+        onUpdateConversationTitle={handleUpdateConversationTitle}
+        isMobileSideBarOpen={isMobileSideBarOpen}
+        isDesktopSideBarOpen={isDesktopSideBarOpen}
+        onMobileCloseClick={() => setIsMobileSideBarOpen(false)}
+        onDesktopCloseClick={() => setIsDesktopSideBarOpen(false)}
```

Comment:
We still need to keep these state variables here because we need to share them with a sibling `NavBar`